From 13f6f9e4c5563339bbad05e1fb9df27a6250119c Mon Sep 17 00:00:00 2001
From: Joakim Zhang <qiangqing.zhang@nxp.com>
Date: Tue, 9 Oct 2018 16:47:45 +0800
Subject: [PATCH 4812/5242] MLK-19692-2 can: flexcan: dts: add wakeup support
 for MX8

commit  51161fc132c2c0d45b11e086027ebe78b21a99d9 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch is to add CAN device into ALIASES list due to i.MX8 series
would analyze wakeup function from firmware(SCU).

Reviewed-by: Dong Aisheng <aisheng.dong@nxp.com>
Reviewed-by: Andy Duan <fugang.duan@nxp.com>
Signed-off-by: Joakim Zhang <qiangqing.zhang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |    3 +++
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    3 +++
 2 files changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index 002e446..e8ba57f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -61,6 +61,9 @@
 		mmc0 = &usdhc1;
 		mmc1 = &usdhc2;
 		mmc2 = &usdhc3;
+		can0 = &flexcan1;
+		can1 = &flexcan2;
+		can2 = &flexcan3;
 	};
 
 	memory@80000000 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index fe1d812..aff0102 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -61,6 +61,9 @@
 		mmc1 = &usdhc2;
 		mmc2 = &usdhc3;
 		usbphy0 = &usbphy1;
+		can0 = &flexcan1;
+		can1 = &flexcan2;
+		can2 = &flexcan3;
 	};
 
 	memory@80000000 {
-- 
1.7.9.5

